testbed/linux/Makefile

```diff
@@ -30,6 +30,7 @@ PLUGINS=\
 	menubar \
 	sample \
 	url_launcher_fde \
+	path_provider_fde \
```

Comment:
Nit: This and the list below are sorted alphabetically to make them easier to manage; please preserve that.